### DISCUSSION TITLE
iPad Application

### DISCUSSION THREAD

**Discussion by Ntebon** [yagiziskirik/Kerbal-Telemetry#14] [Ideas]
Curious to see if there is the potential to make this an app for the game. Back in 2020, I remember someone being interested in BETAs for Kerbal Companion for iOS, but I do not think I ever saw anything come of it.

**Comment by yagiziskirik** [comment]
Since I don't have an Apple developer account, my iOS/iPadOS applications can be used for around a week without reinstalling. But I found a better solution throughout my coding adventures. With a bit of cheating, I can say that it is possible. This feature will be coming in future updates!

![RPReplay_Final1655108777](https://user-images.githubusercontent.com/33431578/173314079-a056a9c1-2246-4305-b1fb-c2afac088592.gif)

But currently, you can use this mod on any mobile device! Start the web server from Kerbal Space Program and from your mobile device, go to Safari, on the address bar enter your device's IP address (here is a [link](https://beebom.com/how-find-ip-address-windows-mac/) for people who don't know how to do it), and start using it. But let me remind you that, the bigger your screen, the more functionality it gets. I hope you will like it!
